packages/shell-api/src/database.ts

```diff
@@ -1427,7 +1427,7 @@ export default class Database extends ShellApiWithMongoClass {
     await this._mongo.setSecondaryOk();
   }
 
-  @serverVersions(['3.1.0', ServerVersions.latest])
+  @serverVersions(['4.4.0', ServerVersions.latest])
```

Comment:
This isn’t accurate, is it? https://docs.mongodb.com/v4.0/reference/method/db.watch/